app/components/visualizer/main-viz/templates/Chart.jsx

```diff
@@ -0,0 +1,42 @@
+import DynamicPieChart from "./DynamicPieChart.js";
+
+class Chart extends React.Component {
+  componentDidMount() {
+    let el = React.findDOMNode(this);
+
+    $.getJSON("blobs/deptGrades.json", (data) => {
+      this.chart = new DynamicPieChart({
+        id: "_id",
+        data: data,
+        element: "#main-viz",
+        height: 512,
+        width: 1024,
```

Comment:
What do you think about this https://blog.safaribooksonline.com/2014/02/17/building-responsible-visualizations-d3-js/